java/src/jmri/jmrix/BooleanPropertyDescriptor.java

```diff
@@ -0,0 +1,24 @@
+package jmri.jmrix;
```

Comment:
This class is in the wrong package and must be moved. The `jmri.jmrix` package is *not* for general NamedBean supporting classes, but for external system connection supporting classes.

Reply:
done.